src/ai/backend/client/cli/session/lifecycle.py

```diff
@@ -613,7 +613,7 @@ def destroy(session_names, forced, owner, stats, recursive):
                     if forced:
                         print_warn(
                             "If you have destroyed a session whose status is one of "
-                            "[`PULLING`, `SCHEDULED`, `PREPARING`, `TERMINATING`, `ERROR`], "
+                            "[`PREPARING`, `TERMINATING`, `ERROR`], "
```

Comment:
The `SCHEDULED` string was not updated last time, right? 
It would be cleaner to have method to manage which states can be destroyed and synchronize them.